jenkins/opensearch/Jenkinsfile

```diff
@@ -77,9 +76,7 @@ pipeline {
                     }
                     steps {
                         script {
-                            buildManifest()
-                            assembleManifest()
-                            uploadArtifacts(upload: true)
+                            buildAndAssemble()
```

Comment:
Should the upload step be different? Not sure if we always want to build and upload especially for someone experimenting with the script. Or change the method name to `buildAndAssembleAndUpload` just to give headsup that this method uploads too

Reply:
Renamed to `buildAssembleUpload`